index.html

```diff
@@ -27,125 +27,144 @@
         id="layoutApp">
 
         <div
-          class="layoutJSON">
+          class="navigationArea">
           <div class="row">
-            <div class="col-xs-9">
-              <input type="checkbox" v-model="kioskMode"/> kioskMode
-              <input type="checkbox" v-model="authorMode"/> authorMode
+            <div class="col-xs-10">
               <button
                 v-for="t in tableaux"
                 class="btn btn-default btn-xs tableau-button"
                 v-bind:class="{ current: isCurrentTableau(t) }"
-                v-on:click="loadTableau(t)">
+                v-bind:disabled="isCurrentTableau(t)"
+                v-on:click="loadTableauByName(t)">
                 {{t}}
               </button>
             </div>
-            <div class="col-xs-3" v-if="!kioskMode">
-              <input type="checkbox" v-model="showSettings"/>&nbsp;Settings
+
+            <div class="col-xs-2">
+              <button
+                class="btn btn-default btn-xs"
+                v-on:click="loadHomeTableau()">
+                Home
+              </button>
+            </div>
+
+            <div class="col-xs-10">
+              <button
+                v-for="item in tableau.layout.cells"
+                class="btn btn-default btn-xs card-button"
+                v-bind:class="{ focused: isCardFocused(item.name) }"
+                v-on:click="toggleFocusOnCard(item.name)">
+                {{item.name}}
+              </button>
             </div>
-            <div class="col-xs-9" v-if="kioskMode&!authorMode">
+            <div class="col-xs-2">
               <button
-                v-for="item in layout"
                 class="btn btn-default btn-xs card-button"
-                v-bind:class="{ focused: isSdTileInFocus(item.contentURL.split('/')[1]) }"
-                v-on:click="focusOnCard(item.contentURL.split('/')[1])">
-                {{item.contentURL.split('/')[1]}}
+                v-on:click="focusOnCard('')">
+                None
               </button>
             </div>
           </div>
-          <div class="row" v-if="showSettings">
+        </div>
+
+        <div
+          v-if="!kioskMode"
+          class="authorArea"
+          @dblclick="toggleAuthorMode()">
```

Comment:
'authorArea' is a bit hidden, it's not easy to figure out double clicking and make it appear/disappear just from the user interface.

Reply:
Yeah. I have a follow up set of mods and it includes a *disclosure triangle* that makes it clearer. See https://bl.ocks.org/DoctorBud/raw/f21853bd0bfb1b8918efd69d921cf128/#Poster for an example. The grid_helper.js file there contains a few fixes beyond this PR... I'll be creating a PR for that soon.